extensions/ffmpeg/src/main/java/com/google/android/exoplayer2/ext/ffmpeg/FfmpegDecoder.java

```diff
@@ -87,7 +87,11 @@ public FfmpegDecoderException decode(DecoderInputBuffer inputBuffer,
     }
     if (!hasOutputFormat) {
       channelCount = ffmpegGetChannelCount(nativeContext);
-      sampleRate = ffmpegGetSampleRate(nativeContext);
+      if ("alac".equals(codecName)) {
```

Comment:
Please submit a ticket and ref it in a comment here. That way someone who encounters this code in the future can see why there's a special case (and update the comment to why if it turns out it was intentional, or remove the special case a few years from now if the ticket has been fixed) :).